src/boot/efi/boot.c

```diff
@@ -1278,6 +1302,54 @@ static void config_defaults_load_from_file(Config *config, char *content) {
                                 }
                                 config->console_mode = u;
                         }
+                } else if (streq8(key, "color-normal-foreground")) {
+                        uint8_t color;
+                        if(!parse_console_color(value, &color)) {
+                                log_error("Unknown color %s", value);
+                        }
```

Comment:
coding style: no {} around single line if blocks. See
https://systemd.io/CODING_STYLE/#formatting

Reply:
I'd like to suggest that the code style should be added to the CI and in some kind of tool that formats on save.

Also thats a 404 link

Reply:
It's on the front page at https://systemd.io/contributing/CODING_STYLE.html.